synapse/storage/databases/main/event_push_actions.py

```diff
@@ -1104,11 +1104,16 @@ def _handle_new_receipts_for_notifs_txn(self, txn: LoggingTransaction) -> bool:
             )
 
             # First ensure that the existing rows have an updated thread_id field.
```

Comment:
Do we also need to fix the `simple_update_many_txn` on line 1274:

https://github.com/matrix-org/synapse/pull/14045/files#diff-f121377d76a7b35c60092da2c4bf8c849544459a2c676cf8e87057aff24ece54R1274-R1282

Reply:
aha, yes